src/java/org/apache/cassandra/net/InboundConnectionInitiator.java

```diff
@@ -460,14 +447,24 @@ void setupMessagingPipeline(InetAddressAndPort from, int useMessagingVersion, in
                         handler.id(true),
                         useMessagingVersion,
                         initiate.framing,
-                        pipeline.get("ssl") != null ? encryptionLogStatement(pipeline.channel(), settings.encryption) : "disabled");
+                        SocketFactory.encryptionConnectionSummary(pipeline.channel()));
 
             pipeline.addLast("deserialize", handler);
 
             pipeline.remove(this);
         }
     }
 
+    private static SslHandler getSslHandler(String description, Channel channel, EncryptionOptions.ServerEncryptionOptions encryptionOptions) throws IOException
+    {
+        final boolean BUILD_TRUST_STORE = true;
```

Comment:
it was a review comment from Dinesh that I agreed with, I think it's about conveying more meaning than a simple boolean rather than rely on the IDE to provide hints what they mean.

Reply:
I'd prefer to keep it as a constant. Not everybody is going to open the code in IntelliJ idea. We follow the Sun Java coding standards with slight modifications. [Constants are to be named all in upper case with underscores as a separator](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html).